test/functional/feature_pop_stratum_compat.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2019 The Bitcoin Core developers
+# Copyright (c) 2019-2020 Xenios SEZC
+# https://www.veriblock.org
+# Distributed under the MIT software license, see the accompanying
+# file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+
+"""
+Start 1 node.
+Run "getblocktemplate" at node[0] height == 0.
+Expect to return no POP fields in JSON response.
+Mine until POP is activated.
+Submit 1 ATV, 1 VTB, 1 VBK (valid) to POP mempool.
+Run "getblocktemplate".
+Expect to get stratum-related fields in response.
+"""
+
+from test_framework.pop import endorse_block, create_endorsed_chain, mine_until_pop_active
+from test_framework.authproxy import JSONRPCException
+from test_framework.pop_const import POP_PAYOUT_DELAY
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    connect_nodes,
+    disconnect_nodes, assert_equal, assert_raises_rpc_error,
+)
+import json
+
+def assert_no_field(source, key):
+    assert isinstance(source, dict)
+    if key in source:
+        raise Exception("Key {} expected to be missing in {}, but it is present: {}".format(key, source, source[key]))
+
+def assert_field_exists(source, key, type):
+    assert isinstance(source, dict)
+    if key not in source:
+        raise Exception("Expected key {} to exist in {}".format(key, source))
+
+    try:
+        if not type(source[key]):
+            raise Exception("Key {} type verification failed. Value={}".format(key, source[key]))
+    except Exception as e:
+        print(e)
+        raise
+
+'''
+If this test fails, then compatibility with Stratum (https://github.com/VeriBlock/vbk-ri-stratum-pool) is broken.
```

Comment:
This test looks like certification test. Is it possible to move it to the alt-integration certification tests?

Reply:
@blackyblack no, because it uses `getblocktemplate` - which is BTC specific

Reply:
We may use a wrapper in vbitcoind_node.py that hides `getblocktemplate` call. For coins unsupported by the VBK Stratum pool, we can exclude this test from test_runner.py.

Reply:
@blackyblack I'd like to merge this as is, then you can propose strategy to copy this test to certification tests